src/components/TransitionLink.js

```diff
@@ -13,6 +13,7 @@ const TransitionLink = ({
   activeStyle,
   style,
   className
```

Comment:
Missing a comma here

Reply:
🤦‍♂️ That is what happens when you rely on the linters too much... Thank you for catching that!